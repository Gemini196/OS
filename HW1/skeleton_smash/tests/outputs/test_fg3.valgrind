==45301== Memcheck, a memory error detector
==45301== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==45301== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==45301== Command: /home/student/Desktop/OS/HW1/skeleton_smash/smash
==45301== Parent PID: 45299
==45301== 
==45301== 
==45301== FILE DESCRIPTORS: 6 open at exit.
==45301== Open file descriptor 1: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg3.out
==45301==    at 0x54EF957: dup2 (syscall-template.S:78)
==45301==    by 0x112CC9: main (smash.cpp:41)
==45301== 
==45301== Open file descriptor 0:
==45301==    at 0x54EF957: dup2 (syscall-template.S:78)
==45301==    by 0x112C77: main (smash.cpp:33)
==45301== 
==45301== Open file descriptor 5:
==45301==    at 0x54EF927: dup (syscall-template.S:78)
==45301==    by 0x112E17: __static_initialization_and_destruction_0(int, int) (smash.cpp:9)
==45301==    by 0x112E33: _GLOBAL__sub_I_stdout_dp (smash.cpp:57)
==45301==    by 0x112EAC: __libc_csu_init (in /home/student/Desktop/OS/HW1/skeleton_smash/smash)
==45301==    by 0x5400C17: (below main) (libc-start.c:266)
==45301== 
==45301== Open file descriptor 4: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg3.out
==45301==    at 0x54EF927: dup (syscall-template.S:78)
==45301==    by 0x112E07: __static_initialization_and_destruction_0(int, int) (smash.cpp:8)
==45301==    by 0x112E33: _GLOBAL__sub_I_stdout_dp (smash.cpp:57)
==45301==    by 0x112EAC: __libc_csu_init (in /home/student/Desktop/OS/HW1/skeleton_smash/smash)
==45301==    by 0x5400C17: (below main) (libc-start.c:266)
==45301== 
==45301== Open file descriptor 3: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg3.valgrind
==45301==    <inherited from parent>
==45301== 
==45301== Open file descriptor 2: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_fg3.err
==45301==    <inherited from parent>
==45301== 
==45301== 
==45301== HEAP SUMMARY:
==45301==     in use at exit: 0 bytes in 0 blocks
==45301==   total heap usage: 59 allocs, 59 frees, 85,119 bytes allocated
==45301== 
==45301== All heap blocks were freed -- no leaks are possible
==45301== 
==45301== For counts of detected and suppressed errors, rerun with: -v
==45301== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
