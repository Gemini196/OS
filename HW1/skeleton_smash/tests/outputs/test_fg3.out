smash> smash> smash> smash> smash> smash> sleep 300& : 46963
smash: got ctrl-C
smash: process 46963 was killed
smash> [1] sleep 100& : 46961 1 secs
[2] sleep 200& : 46962 1 secs
[4] sleep 400& : 46964 1 secs
[5] sleep 500& : 46965 1 secs
smash> sleep 100& : 46961
smash: got ctrl-C
smash: process 46961 was killed
smash> [2] sleep 200& : 46962 1 secs
[4] sleep 400& : 46964 1 secs
[5] sleep 500& : 46965 1 secs
smash> sleep 200& : 46962
smash: got ctrl-C
smash: process 46962 was killed
smash> [4] sleep 400& : 46964 2 secs
[5] sleep 500& : 46965 2 secs
smash> sleep 500& : 46965
smash: got ctrl-C
smash: process 46965 was killed
smash> [4] sleep 400& : 46964 2 secs
smash> sleep 400& : 46964
smash: got ctrl-C
smash: process 46964 was killed
smash> smash> smash: sending SIGKILL signal to 0 jobs:
